e2e/nomostest/client.go

```diff
@@ -139,6 +144,9 @@ func upsertCluster(t testing.NTB, cluster Cluster, opts *ntopts.New) (bool, erro
 	if err := cluster.Create(); err != nil {
 		return true, err
 	}
+	if err := cluster.WaitForReady(); err != nil {
```

Comment:
Is there evidence that suggests this is needed? We currently wait for all operations on the cluster to complete during the Create call above

Reply:
Just ran the tests locally on GKE and ran into the same issue where this doesn't wait for the cluster to be ready. This wasn't an issue before, perhaps there was a change to the GKE operations during cluster creation.